shynet/core/migrations/0009_auto_20220624_0744.py

```diff
@@ -0,0 +1,90 @@
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0008_auto_20200628_1403'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='service',
+            options={'ordering': ['name', 'uuid'], 'verbose_name': 'Service', 'verbose_name_plural': 'Services'},
+        ),
+        migrations.AlterField(
+            model_name='service',
+            name='collaborators',
+            field=models.ManyToManyField(blank=True, related_name='collaborating_services', to=settings.AUTH_USER_MODEL, verbose_name='Collaborators'),
+        ),
+        migrations.AlterField(
+            model_name='service',
+            name='collect_ips',
+            field=models.BooleanField(default=True, verbose_name='Collect ips'),
+        ),
+        migrations.AlterField(
+            model_name='service',
+            name='created',
+            field=models.DateTimeField(auto_now_add=True, verbose_name='created'),
+        ),
+        migrations.AlterField(
+            model_name='service',
+            name='hide_referrer_regex',
+            field=models.TextField(blank=True, default='', validators=[core.models._validate_regex], verbose_name='Hide referrer regex'),
+        ),
+        migrations.AlterField(
+            model_name='service',
+            name='ignore_robots',
+            field=models.BooleanField(default=False, verbose_name='Ignore robots'),
+        ),
+        migrations.AlterField(
+            model_name='service',
+            name='ignored_ips',
+            field=models.TextField(blank=True, default='', validators=[core.models._validate_network_list], verbose_name='Igored ips'),
+        ),
+        migrations.AlterField(
+            model_name='service',
+            name='link',
+            field=models.URLField(blank=True, verbose_name='link'),
+        ),
+        migrations.AlterField(
+            model_name='service',
+            name='name',
+            field=models.TextField(max_length=64, verbose_name='Name'),
+        ),
+        migrations.AlterField(
+            model_name='service',
+            name='origins',
+            field=models.TextField(default='*', verbose_name='origins'),
+        ),
+        migrations.AlterField(
+            model_name='service',
+            name='owner',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='owning_services', to=settings.AUTH_USER_MODEL, verbose_name='Owner'),
+        ),
+        migrations.AlterField(
+            model_name='service',
+            name='respect_dnt',
+            field=models.BooleanField(default=True, verbose_name='Respect dnt'),
+        ),
+        migrations.AlterField(
+            model_name='service',
+            name='script_inject',
+            field=models.TextField(blank=True, default='', verbose_name='Script inject'),
+        ),
+        migrations.AlterField(
+            model_name='service',
```

Comment:
I don't think that these are really needed, but it's common practice in Django to create a migration if - like i did - a verbose_name field is added to the model fields. You can find a more detailed explanation here: https://stackoverflow.com/questions/58555007/does-django-create-useless-migrations

It seems like there are ways to avoid these migrations, but at the moment i don't see why we should do that. 
Let me know, what you think about it...